lib/ddtrace/manual_tracing.rb

```diff
@@ -8,15 +8,15 @@ class << self
       # TODO: Assumes it will have access to the context from the span.
       #       This won't be true in the future. Will need to change this.
       def keep(span_op)
```

Comment:
Do we need to keep this module still?

Reply:
I would *love* to delete this. And I think I will; its superfluous and of bad design. Spans shouldn't have knowledge or means of modifying the trace... it's very indirect.